fix(gamelogic): correct beetle-move validation

Yasamato opened the pull request:
Added missing offset of 1 piece (beetle itself) at from-tile. #99

Yasamato committed:
fix(gamelogic): corrected beetle-move validation

Added missing offset of 1 piece (beetle itself) at from-tile. #99

Yasamato committed:
fix(error): corrected error-msg display

Xerus2000 commented:
Bitte nutz den ["fixes"-syntax](https://help.github.com/en/github/managing-your-work-on-github/closing-issues-using-keywords) ;)

Xerus2000 commented:
und was ist "error" für ein scope?

Yasamato committed:
chore(text): improved error-msg

This commit improves the error-message on server-errors after a move has been sent. #101

Yasamato committed:
chore(dialog): optimized error-handling

This commit adds an automated game-reload to display a proper server response on error. Furthermore the communication between render and main process has been enhanced.
